### PR TITLE
fix(nd): Waypoint Icon Size, Icon Outlines, Airplane Icon Color and Outline

### PR DESCRIPTION
**Pull request by marcman86**
Fixes #[issue_no]

## Summary of Changes

- Now using correct Font again.
- Waypoint size
- Icons Outlines (Airports, VOR, Decel)
- Airplane Icon Color and Outline

## References

image

## Additional context

FS uses .png icons exported from .svg files. All original .svgs are still there. I copied and renamed them for my change. The .pngs are overwritten.

Discord username (if different from GitHub):

## Testing instructions

Test ND icons. Think they look better now. Was not able to change font colors for Airports and VORs cause then also the waypoint font color changes. So they are still all green.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by tyler58546** [comment]
In your screenshots it looks like the waypoint labels have the wrong font

**Comment by MisterChocker** [comment]
> In your screenshots it looks like the waypoint labels have the wrong font

AFAIK The fonts have not been subject to change in this pr. So they must have been this way before?

**Comment by marcman86** [comment]
I know fonts were changed in #3498. But its often gets overprinted.

**Comment by marcman86** [comment]
More rounded Airplane Icon

**Comment by MisterChocker** [comment]
@tyler58546 mapConfig.json still has `ECAMFontRegular` as `waypointLabelFontFamily`. Or is there some place else? Could it have been overridden by the hard-fork?

**Comment by marcman86** [comment]
A32NX-NavSystem has Roboto in it.

**Comment by MisterChocker** [comment]
Fixed Font 
image

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.css [hunk 1]
```diff
@@ -1,18 +1,6 @@
 ﻿@font-face {
-  font-family: "Roboto";
-  src: url("/Fonts/RobotoMono-Medium.ttf") format("truetype");
+  font-family: "Liberation Mono";
+  src: url("/Fonts/LiberationMono.ttf") format("truetype");
```
Comment (tyler58546) [inline flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.css:3]: Shouldn't this be `ECAMFontRegular`?

**Comment by marcman86** [comment]
With ECAMFontRegular
grafik

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.css (hunk 1 above)
Comment (MisterChocker) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.css]: yup its changed now.

**Comment by Giuse101** [comment]
I agree. The fonts are not correct.

**Review by tyler58546 — APPROVED** [review]
LGTM

**Comment by marcman86** [comment]
Forget to rebuild. In the PR its fine. Here an image:
grafik

From my side this is ready for review and to test :-)

**Comment by TheSquirtingPantin** [comment]
Category: QA Trial
Name: Discord User Rémi#8871
Date of testing: Current date 13/04/2021
Version of the sim: V 1.14.6.0
PR Tested: #4385
Tier of testing: 1
Changes to observe:
New font, waypoint size and airplane icon on ND 

Testing technique:
1. Full flight from KDEN to KIAD
2. Visual check for new airplane icon, new font and waypoint size.
3.
4.

Flight notes (When done a full flight):
KDEN/17R EMMYS7 ZIRKL DCT PWE DCT SPI J80 VHP DCT APE DCT MGW GIBBZ4 KIAD/01R

Media: N/A

Issues (If Any): N/A


Overall Rating: Good
Conclusions of the testing: LGTM, good job !

**Comment by kberg93** [comment]
Category: QA Tester
Name: Kberg #0666
Date of testing: 16/04/2021
Version of the sim: 1.15.7.0
PR Tested: #4385 
Tier of testing: 2
Changes to observe:
- Now using correct Font again.
- Waypoint size
- Icons Outlines (Airports, VOR, Decel)
- Airplane Icon Color and Outline

Testing technique: 
Steps you've made to test the Pull Request:
1. C&D at ENSB all went well, just the ND "bug" being ther (see attatched video).
2. Take-off and climb to crz alt no issues fonts etc looks good.
3. Mid crz ND works as normal (magic is real).
4. Landed and all well at ENTC.

Flight notes (When done a full flight): ENSB AMPI1A AMPIS P853 LOMVI LOMV3M ENTC

Media: 
FlightSimulator_Xh2TNekaxL


https://user-images.githubusercontent.com/56963782/115080249-b75cdf00-9f02-11eb-9d5a-ca42c7d628b2.mp4


Issues (If Any): Not sure if its related to this PR or not but just adding a clip of what i found strange (fixed itself midflight).
Overall Rating: Good.
Conclusions of the testing: As far as i could see everything looked like its supposed to be. _Just check if someone else gets same ish issue or not, its 100% repeatable for me 5/5 times setting up flightplan the same way._

**Comment by marcman86** [comment]
Hey, thanks for the testing so far. I found a few things to make this PR cleaner and more complete. So i will close it and reopen it.

**Comment by MisterChocker** [comment]
You don't need to close this pr. You can simply update it as you work along.
Simply convert it to Draft so people know you are still working on it.

**Comment by marcman86** [comment]
Sorry, also saw this now. Didn't found the draft option.

**Comment by MisterChocker** [comment]
image

**Pull request closed without merging**